packages/nextcloud/lib/src/version_supported.dart

```diff
@@ -1,33 +1,57 @@
 part of '../nextcloud.dart';
 
+/// Version of core/Server supported
+const coreSupportedVersion = 26;
+
+/// API version of the news app supported
+const newsSupportedVersion = 'v1-3';
+
+/// API version of the notes app supported
+const notesSupportedVersion = 1;
+
+/// All supported versions
+const supportedVersions = [
+  coreSupportedVersion,
+  newsSupportedVersion,
+  notesSupportedVersion,
+];
+
 // ignore: public_member_api_docs
 extension CoreVersionSupported on NextcloudCoreClient {
   /// Checks if the app on the server is supported by the client
```

Comment:
I know I'm getting nit picky here but the doc comments also need adjustments

Reply:
What do you want to have here instead?

Reply:
I'd mention that it what it returns.
How should I know what the second parameter represents when just using code highlighting in an IDE.
 
 sadly I couldn't find anything in the effective-dart:documentation guide regarding this issue.
 ```dart
 /// returns something cool
(bool issupported, int minVersion) checkSomething() {

}
 ```
 
 Also the IDE does not really highlight the names I gave it in the above example o the best solution might be adding a 'Returns the result of the check and the minimum supported version.'
 
 Or maybe a code example :thinking: